cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Conway/DRepRetirement.hs

```diff
@@ -7,6 +7,7 @@
 
 module Cardano.Testnet.Test.Cli.Conway.DRepRetirement
   ( hprop_drep_retirement
+  , waitDRepsNumber
```

Comment:
I don't think we should create dependencies between test modules. Can you move this to `src/` somewhere in `cardano-testnet`? I think also this become useful.

Additionally, if the function does the check, I suggest using `check` prefix to indicate possible failure.